Development/nmos-cpp-node/config.json

```diff
@@ -282,5 +289,79 @@
+    //"authorization_request_max": 30,
+
+    // fetch_authorization_public_keys_interval_min/fetch_authorization_public_keys_interval_max [registry, node]: used to poll for Authorization API public keys changes; default is about one hour
+    // "Resource Servers (Nodes) SHOULD seek to fetch public keys from the Authorization Server at least once every hour. Resource Servers MUST vary their retrieval
+    // interval at random by up to at least one minute to avoid overloading the Authorization Server due to Resource Servers synchronising their retrieval time."
+    // See https://specs.amwa.tv/is-10/releases/v1.0.0/docs/4.1._Behaviour_-_Authorization_Servers.html#authorization-server-public-keys
+    //"fetch_authorization_public_keys_interval_min": 3600,
+    //"fetch_authorization_public_keys_interval_max": 3660,
+
+    // access_token_refresh_interval [node]: time interval (in seconds) to refresh access token from Authorization Server
+    // It specified the access token refresh period otherwise Bearer token's expires_in is used instead.
+    // See https://specs.amwa.tv/is-10/releases/v1.0.0/docs/4.4._Behaviour_-_Access_Tokens.html#access-token-lifetime
+    //"access_token_refresh_interval": -1,
+
+    // client_authorization [node]: whether clients should use authorization to access protected APIs
+    //"client_authorization": false,
+
+    // server_authorization [registry, node]: whether server should use authorization to protect its APIs
+    //"server_authorization": false,
+
+    // authorization_code_flow_max [node]: timeout for the authorization code workflow (in seconds)
+    // No timeout if value is set to -1, default to 30 seconds
+    //"authorization_code_flow_max": 30,
+
+    // authorization_flow [node]: used to specify the authorization flow for the registered scopes
+    // supported flow are authorization_code and client_credentials
+    // client_credentials SHOULD only be used for NO user interface node, otherwise authorization_code MUST be used
+    //"authorization_flow": "authorization_code",
+
+    // authorization_redirect_port [node]: redirect URL port for listening authorization code, used for client registration
+    //"authorization_redirect_port": 3218,
+
+    // initial_access_token [node]: initial access token giving access to the client registration endpoint for non-opened registration
+    //"initial_access_token", "",
+
+    // authorization_scopes [node]: used to specify the supported scopes for client registration
+    // supported scopes are registration, query, node, connection, events and channelmapping
+    //"authorization_scopes": array-of-scope-string,
+
+    // token_endpoint_auth_method [node]: String indicator of the requested authentication method for the token endpoint
+    // supported methods are none, client_secret_basic and private_key_jwt, default to client_secret_basic, where none is used for public client
+    //"token_endpoint_auth_method": "client_secret_basic",
```

Comment:
Is it worth mentioning that jwk is generated from the TLS cert, so can only be used when TLS is configured?